### DISCUSSION TITLE
Horizontal Brackets

### DISCUSSION THREAD

**Discussion by danielwdunn** [aderusha/DDD-Printable-Wall-Control-System#20] [Q&A]
My system is mounted with the holes oriented horizontally. 

Which base models should I use in addition to the horizontal models? Are there any horizontal quickhooks?

**Comment by aderusha** [comment]
Wall Control sells panels that are either vertically or horizontally aligned, and the hole patterns they use in those panels are different.  Here's what that looks like:

![Vertical_vs_Horizontal_Wall_Control_Patterns](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/images/Vertical_vs_Horizontal_Wall_Control_Patterns.png?raw=true)

The only place where this matters will be for models which engage with the holes, which would be any model that utilizes the locking feature.  The only place you'll see much of this is the locking center spacers, where [this folder](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Locking_spacer) has locking spacers for vertical panel and [this folder](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Locking_spacer_for_horizontal_Wall_Control) has locking spacers for horizontal panels.

The Quickhook models don't use the locking pin, so they are all OK.
